thundra/thundra_agent.py

```diff
@@ -60,6 +61,8 @@ def __init__(self,
             # Pass thundra instance to integration for wrapping handler wrappers
             handler_wrappers.patch_modules(self)
 
+        self.initialize_debugger()
```

Comment:
I think, we should call `initialize_debugger` only if debugging enabled. As far as I see, in all cases in tries to import `ptvsd` and prints error message if it is not available from layer